pyne/partisn.py

```diff
@@ -822,17 +759,19 @@ def strip_mat_name(mat_name):
     'mat:' or special characters.
     Assumes PyNE naming convention (must start with 'mat:').
     """
+
+    # Remove "mat:"
+    mat_name = mat_name.split('mat:')[1]
+
+    # Replace "/rho" with an underscore
+    mat_name = string.replace(mat_name, "/rho:", "_")
     
-    # Remove 'mat:'
-    tmp1 = mat_name.split(':')[1]
-    
-    # Remove other special characters
-    special_char = [':', ',', ' ', ';', "'", '"']
+    # Remove any other special characters
+    special_char = [':', ',', ' ', ';', "'", '"', '/']
```

Comment:
I think you should do something more robust like, remove all non alphanumeric characters, for example soon we will add the '@' operator, which I expect would break this. Something like http://stackoverflow.com/a/1276804